### PR TITLE
Azure example terraform

### PR DESCRIPTION
**Pull request by jklegar**
**What this PR does / why we need it**: An example terraform configuration to run Feast on Azure

**Which issue(s) this PR fixes**:

Fixes #

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: infra/terraform/azure/README.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
+# Terraform config for Feast on Azure
+
+This serves as a guide on how to deploy Feast on Azure. At the end of this guide, we will have provisioned:
+1. AKS cluster
+2. Feast services running on AKS
+3. Azure Cache (Redis) as online store
+4. Spark operator on AKS
+5. Kafka running on HDInsight.
```
Comment (woop) [inline infra/terraform/azure/README.md:8]: By the way, is there a reason we are using HDInsights over Event Hubs for Kafka?

**Review by woop — APPROVED** [review]

File: infra/terraform/azure/README.md (hunk 1 above)
Comment (jklegar) [reply infra/terraform/azure/README.md:8]: HDInsight actually runs Kafka (it's essentially Kafka managed by Azure), whereas Event Hubs is Kafka-client-compatible but doesn't support certain Kafka features. Seemed faster and more likely to work using HDInsight

**Pull request merged**